docs/sources/release-notes/v3.0.md

```diff
@@ -0,0 +1,63 @@
+---
+title: v3.0
+description:  Version 3.0 release notes.
+weight: 30
+---
+
+# v3.0
+
+Grafana Labs and the Loki team are excited to announce the release of Loki 3.0. Here's a summary of new enhancements and important fixes.
+
+For a full list of all changes and fixes, refer to the [CHANGELOG](https://github.com/grafana/loki/blob/release-3.0.0-rc1/CHANGELOG.md).
+
+## Features and enhancements
+
+Key features in Loki 3.0.0 include the following:
+
+- **Query acceleration with Bloom filters** (experimental): This is designed to speed up filter queries, with best results for queries that are looking for a specific text string like an error message or UUID.
```

Comment:
I think we should link to https://github.com/grafana/loki/pull/12486/files#diff-6e3d065261f76ec383ca4882ced2e787195baa3a407eeb8403cd014e689b8235 when we get this to publish